clock/monotonic/monotonic.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package monotonic
+
+import (
+	"time"
+	_ "unsafe"
+)
+
+//go:noescape
+//go:linkname nanotime runtime.nanotime
```

Comment:
I'm very wary of depending on runtime internals like this. We only *need* it on Linux, so why not just use the syscall?